### DISCUSSION TITLE
如何禁用 ` grave 的筆畫功能? 或者如何自定义笔画功能的快捷键?

### DISCUSSION THREAD

**Discussion by iugo** [rime/home#723] [Q&A]
目前, 单击 grave 键会出现笔画功能:

```
`〔筆畫〕
```
<img width="145" alt="截屏2021-01-25 上午1 54 17" src="https://user-images.githubusercontent.com/499364/105638886-3ddbc480-5eb0-11eb-91b0-4b755ba66e13.png">

这个功能很好, 但是对于编程来说, grave accent (U+0060) 是很常用的一个符号, 总是通过 return 按键去再次确认很麻烦. 能否自定义这一按键呢?

**Comment by iugo** [comment]
在这里找到了禁用的例子, 如:

https://github.com/rime/home/wiki/CustomizationGuide#%E6%B4%BB%E7%94%A8%E6%A8%99%E9%BB%9E%E5%89%B5%E5%BB%BA%E8%87%AA%E5%AE%9A%E7%BE%A9%E8%A9%9E%E7%B5%84

但没有找到更换快捷键的完整例子.

如果可以通过别的快捷键进行触发, 那就完美了.

**Comment by yanhuacuo** [comment]
在相应的 schema 下，有 「recognizer/patterns」，「笔画」项绑定了「grave」键。你懂正则，一看就明白。
而且该「笔画」项，在「segmentors」下挂了单。改的话，改「recognizer/patterns」下的正则，删的话，「recognizer/patterns」下的正则项与「segmentors」下的挂单项一块清掉就可以了。

**Comment by lotem** [comment]
​相關代碼行：
https://github.com/rime/rime-luna-pinyin/blob/19726ceca1b343d660382903d100f517bdc775ab/luna_pinyin.schema.yaml#L50
https://github.com/rime/rime-luna-pinyin/blob/19726ceca1b343d660382903d100f517bdc775ab/luna_pinyin.schema.yaml#L83
https://github.com/rime/rime-luna-pinyin/blob/19726ceca1b343d660382903d100f517bdc775ab/luna_pinyin.schema.yaml#L101
